src/NHibernate.Test/CfgTest/ConfigurationSchemaFixture.cs

```diff
@@ -27,6 +28,8 @@ public void InvalidConfig()
 		[Test]
 		public void FromAppConfigTest()
 		{
+			Assume.That(Assembly.GetEntryAssembly()?.GetName().Name != "testhost");
```

Comment:
This is used at many place. A common helper for testing this could be nice (and its naming would be more explicit about what case it detects than this code). (I guess this is about the app.config issue in tests.)

Reply:
Done